### ISSUE TITLE
📃: HandWriting Recognition using Deep learning

### ISSUE THREAD

**Issue by Nav3005** [Avdhesh-Varshney/AI-Code#112]
:red_circle: **Title** : Handwriting Recognition system using Tensorflow
:red_circle: **Aim** : To develop an intelligent handwriting recognition system using TensorFlow that accurately interprets handwritten text 
:red_circle: **Brief Explanation** : This project aims to implement a handwriting recognition system using deep learning techniques and utilized to create and train a Convolutional Neural Network (CNN) capable of recognizing numbers.

### Screenshots 📷

N/A

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Naveen karthik Rajakumar
- What is your participant role? SWOC

***********************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by adityasingh-0803** [comment]
even i want to work for this

**Comment by 11PRIMUS** [comment]
@adityasingh-0803  why don't you work on some other image recognition (like cancer classification, seed or facial expression classification)

**Comment by Avdhesh-Varshney** [comment]
@Nav3005 What's the update?

**Comment by Nav3005** [comment]
@Avdhesh-Varshney Ive been asked to add atleast models in my previous PR for this project and im still working on it

**Comment by Avdhesh-Varshney** [comment]
@Nav3005 OK. Solve it as soon as possible as time is already finished. 👍

**Comment by 11PRIMUS** [comment]
@Nav3005 make sure to complete it by tomorrow
